src/aat/resources/cucumber/answers.feature

```diff
@@ -19,7 +19,23 @@ Feature: Answers
     And the response headers contains a location to the created entity
     And send get request to the location
     And the response code is 200
-    And the notification scheduler runs
+    And wait until the event is processed
+    And there is no event queued for this online hearing of event type answers_submitted
```

Comment:
This is a bit odd because it's testing a negative. I couldn't think of another way to prove that a drafted answer does not result in an event being queued.